opentelemetry-sdk/tests/metrics/export/test_export.py

```diff
@@ -49,7 +50,7 @@ def test_export(self):
         )
         labels = {"environment": "staging"}
         aggregator = SumAggregator()
-        record = MetricRecord(metric, labels, aggregator)
+        record = MetricRecord(metric, labels, aggregator, meter.resource)
```

Comment:
I think it would be good to have a test that tests to see if the resource created in the metric record matches the resource in the meterprovider.

Reply:
Updated the ConsoleMetricsExporter to show the resource and updated the test to validate it.

Reply:
I think it would be better to remove `resource` from `Meter`: https://github.com/codeboten/opentelemetry-python/pull/9

Reply:
The spec does specify that a `Meter` should bind an API with a resource: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/sdk.md#sdk-terminology, but I guess since we're not making use of the resource yet, it's ok to remove it here for now.